gcapi/gcapi.py

```diff
@@ -297,6 +297,28 @@ def for_image(self, pk):
         return result
 
 
+class RetinaPolygonAnnotationSetsAPI(APIBase, ModifiableMixin):
+    base_path = "retina/polygon-annotation-set/"
+    json_schema = import_json_schema("polygon-annotation.json")
+    modify_json_schema = import_json_schema("post-polygon-annotation.json")
+
+    def for_image(self, pk):
+        result = self._client(
+            method="GET", path=self.base_path, params={"image": pk}
+        )
+        for i in result:
+            self._verify_against_schema(i)
+        return result
```

Comment:
This won't work if there is more than 1 page, right? Why have a specific method for this rather than letting the user provide the params?

Reply:
Pagination is disabled on grand challenge for this endpoint or is that not what you mean?
Anyway, the method is indeed not necessary I can just pass the params in cirrus-core to the `list()` method, so I removed it.

Reply:
Pagination should be enabled by default, but if you make a get request to a list you will only ever get 100 results, then you will need to iterate over `next=`.